tycho-its/projects/target.references/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- - Copyright (c) 2015, 2020 SAP SE and others. - All rights reserved. 
+	This program and the accompanying materials - are made available under the 
+	terms of the Eclipse Public License v1.0 - which accompanies this distribution, 
+	and is available at - http://www.eclipse.org/legal/epl-v10.html - - Contributors: 
+	- SAP SE - initial API and implementation -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.eclipse.tycho.itests</groupId>
+	<artifactId>issue-401-parent</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>pom</packaging>
+
+	<modules>
+		<module>test.bundle</module>
+		<module>target.refs</module>
+	</modules>
+
+	<properties>
+		<tycho-version>2.6.0-SNAPSHOT</tycho-version>
```

Comment:
I still haven't understood why this is defined again and again in many ITs (and with older versions than 2.6 in some cases). Shouldn't all tests point to the current development version for testing (by inheriting from the root parent of tycho)? What am I missing there?

Reply:
The` tycho-version` is overridden in the actual i-test run with the current build version so in that value is then irrelevant in this case as it is never used.
But if you are open the pom.xml in eclipse m2e of course complains that `tycho-version` is undefined and all kind of error markers are shown and code-completion does not work. Thus it is defined in (some but not all) of the itest mostly with the version that was recent at time of writing the test, it can always be adjusted to the latest version of course.

Just for convenience, it allows to right-click and Choose `Run As > Maven` install to test without a full itest run or specifying the version on the commandline.